workflows/edit_eml/edit_semantic_annotation.Rmd

```diff
@@ -39,17 +55,68 @@ $valueURI$valueURI
 ```
 
-### Ontologies used
+### How to add an annotation
+
+**1. Decide which variable to annotate**
+
+The goal for the datateam is to start annotating every dataset that comes in. Practically, we cannot annotate
+every single variable in every single dataset - this would take forever! Typically, there are a few key variables
+that are measured in any particular dataset. These key variables are the ones we want to prioritize. Although having annotations on 
+variables like date, longitude, or site name would be helpful, they shouldn't be our primary focus.
+
+
+**2. Find an appropriate `valueURI`**
+
+The next step is to find an appropriate value to fill in the blank of the sentence: "this attribute contains measurements of _____."
+
+There are several ontologies to search in. In order of most to least likely to be relevant to the Arctic Data Center they are:
+
+* [The Ecosystem Ontology (ECSO)](http://bioportal.bioontology.org/ontologies/ECSO/?p=classes&conceptid=http://purl.dataone.org/odo/ECSO_00002925)  
+    - this was developed at NCEAS, and has many terms that are relevant to ecosystem processes, especially those involving carbon and nutrient cycling
+* [The Environment Ontology (EnVO)](http://bioportal.bioontology.org/ontologies/ENVO/?p=classes&conceptid=root)
+    - this is an ontology for the concise, controlled description of environments
+* [National Center for Biotechnology Information (NCBI) Organismal Classification (NCBITAXON)](http://bioportal.bioontology.org/ontologies/NCBITAXON/?p=classes&conceptid=root)
+    - The NCBI Taxonomy Database is a curated classification and nomenclature for all of the organisms in the public sequence databases.
+* [Information Artifact Ontology (IAO)](http://bioportal.bioontology.org/ontologies/IAO/?p=summary)
+    - this ontology contains terms related to information entities (eg: journals, articles, datasets, identifiers)
+
+To search, navigate through the "classes" until you find an appropriate term.
+
+**3. Build the annotation in R**
+
+We will have helper functions for this in the coming months, but for now it will be a bit of a manual process.
+
+First you need to figure out the index of the attribute you want to annotate.
+
+```{r, eval = F}
+eml_get_simple(doc$dataset$dataTable[[3]]$attributeList, "attributeName")
+```
+
+```
+ [1] "prdM"         "t090C"        "t190C"        "c0mS/cm"      "c1mS/cm"      "sal00"        "sal11"        "sbeox0V"      "flECO-AFL"
+[10] "CStarTr0"     "cpar"         "v0"           "v4"           "v6"           "v7"           "svCM"         "altM"         "depSM"    
+[19] "scan"         "sbeox0ML/L"   "sbeox0dOV/dT" "flag"    
+```
+
+Next, assign an `id` to the attribute. It should be unique within the document, and it's nice if it is human readable and related to the attribute it is describing.
 
-#### OBOE: The Extensible Observation Ontology
+```{r, eval = F}
+doc$dataset$dataTable[[3]]$attributeList$attribute[[6]]$id <- "salinity_var"
```

Comment:
I coulda provided an even better example if I was thinking more about this earlier. For attribute `id`s, sometimes we use a pattern of `entity_x_attribute_y` to give the attribute scope while recognizing attribute names are often repeated across entities. So a file called `diet.csv` with an attribute `species` might have an `id` of `diet_species`. Simple example that may not apply well in real life but thought I'd share.